srv/pom.xml

```diff
@@ -68,6 +68,11 @@
 			<artifactId>scp-cf</artifactId>
```

Comment:
> Module com.sap.cloud.sdk.cloudplatform:scp-cf has been deprecated and functionally replaced by com.sap.cloud.sdk:sdk-core

https://sap.github.io/cloud-sdk/docs/java/guides/5.0-upgrade-steps

Reply:
Makes sense, though the old module would still be there.